models/stg_linkedin__ad_analytics_by_creative.sql

```diff
@@ -26,6 +26,11 @@ with base as (
         {% else %}
         cost_in_usd as cost
         {% endif %}
+
+        {% if var('linkedin__passthrough_metrics', []) != [] %}
```

Comment:
Question for my curiosity: 

Do we need to add `,[]` and `!= []` to the conditional? Would the below also work?
```
{% if var('linkedin__passthrough_metrics') %}
      , {{ var('linkedin__passthrough_metrics') | join (", ")}}
{% endif %}
```
I have seen us use both types and I think they both result in the same outcome, but wanted to get your thoughts as well.

Reply:
yeah i don't think we need to add it, particularly since we give that variable a default value of `[]` in the package dbt_project.yml file. i'll remove!